starknet/src/utils/single_slot_proof.cairo

```diff
@@ -11,7 +11,51 @@ mod SingleSlotProof {
     #[storage]
     struct Storage {
         _timestamp_remappers: ContractAddress,
```

Comment:
do you know why it's `remapperS` if it's a single contract? is it a contract consisting of multiple contracts?

Reply:
there can technical me multiple mappings in the contract i think, not sure when that would be useful though.